### PR TITLE
Add fade-in animation to drawer dimmer

### PR DESCRIPTION
**Pull request by grosbouddha**
The new animation spec (300ms) comes from the v3/dashvboard dimmer (confimed with Nico Duval).

### CODE REVIEW

File: src/drawer/style.ts [hunk 1]
```diff
@@ -15,20 +15,20 @@ export default css`
...
     bottom: 0;
     right: 0;
     z-index: 1;
-    background-color: rgba(0, 0, 0, 0);
-    transition: background-color 150ms linear;
-  }
-
-  .kirk-drawer--open::after {
     background-color: rgba(0, 0, 0, 0.3);
+    transition: background-color 300ms linear;
```
Comment (Bidjii) [inline src/drawer/style.ts]: If I may intervene here, should this `300ms` duration go in the `branding` file?

Comment (nickoola) [reply src/drawer/style.ts]: We already have `transition: background-color ${transition.duration.fast} linear;` for example but it setted to `200ms`.

Comment (wakooka) [reply src/drawer/style.ts]: Yes can you use one of the value in the branding file? Or if it does'n fit your case add a new one
https://github.com/blablacar/ui-library/blob/master/src/_utils/branding.ts#L82

Comment (grosbouddha) [reply src/drawer/style.ts]: Updated to ${transition.duration.base} , thanks!

**Comment by grosbouddha** [comment]
PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,8 @@
 - **[UPDATE]** Removing css linter (legacy, targeting CSS files which does not exist anymore on this repository)
 - **[UPDATE]** Moving to styled-jsx V3
 - **[UPDATE]** Re-organize the dependencies (dev & prod dependencies)
+- **[FIX]** Fix fade-in animation for Drawer dimmer
```
Comment (wakooka) [inline CHANGELOG.md]: Move this to `Unreleased` section

Comment (grosbouddha) [reply CHANGELOG.md]: Done

**Pull request merged**